### PR TITLE
Dependencies, starting with Hapi and sample code. Updated the readme …

### PR DESCRIPTION
**Pull request by eroepken**
…to include install/run instructions after adding Hapi.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "type": "git",
+    "url": "git+https://github.com/mealeyst/TaurusCMS.git"
+  },
+  "keywords": [
+    "node",
+    "cms",
+    "js",
+    "taurus"
+  ],
+  "author": "Jason de Oliveira, Ben Helmer, Shawn Mealey and Ellie Strejlau",
```
Comment (eroepken) [inline package.json]: I did this in last name ABC order. If this should change, let me know, or feel free to change it in future commits.

Comment (jpdeoliveira) [inline package.json]: I think the order should be "Shawn Mealey, Ellie Strejlau, Ben Helmer and Jason de Oliveira." In the least, Shawn and Ellie should be first.

Comment (mealeyst) [inline package.json]: So @eroepken if you want to you can add yourself as the author and add us as contributors. This setup ideally should look like:
```
"author": "Shawn Mealey <shawn@flywheelsports.com>",
"contributors": [
    {
      "name": "Michael Burlando",
      "email": "Michael@flywheelsports.com"
    },
    {
      "name": "Jonathan Williams",
      "email": "JWilliams@flywheelsports.com"
    }
  ],
]
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "tauruscms",
+  "version": "1.0.0",
```
Comment (mealeyst) [inline package.json]: Since this is in alpha should the version be 0.0.0?

File: package.json (hunk 1 above)
Comment (eroepken) [inline package.json]: Truth. I will change the formatting in a few minutes and re-commit.

**Comment by mealeyst** [comment]
All in all this PR looks pretty good. I might also consider that we install mocha, istanbul, and babel. ` npm install --save-dev mocha istanbul babel ` This should still keep our tech stack pretty small.

If you agree with the suggestions I made and implement them, consider this PR to have a :+1: from me and feel free to merge it in. If you want to discuss more, I am down for that too.

**Comment by eroepken** [comment]
Oops. There's a syntax error in the .json file. Fixing this now.

**Comment by eroepken** [comment]
Rebased and ready for review again.

**Pull request merged**